src/applications/_mock-form-ae-design-patterns/components/FormPages/VeteranProfileInformation.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import moment from 'moment';
```

Comment:
If possible can this be updated at some point to use `date-fns`? 

https://dsva.slack.com/archives/C5HP4GN3F/p1721065353985949